<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QWidget" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>214</height>
   </rect>
  </property>
  <property name="palette">
  <palette>
   <active>
    <colorrole role="WindowText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Button">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>46</red>
       <green>52</green>
       <blue>54</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Light">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>69</red>
       <green>78</green>
       <blue>81</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Midlight">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>57</red>
       <green>65</green>
       <blue>67</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Dark">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>23</red>
       <green>26</green>
       <blue>27</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Mid">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>30</red>
       <green>34</green>
       <blue>36</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Text">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="BrightText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="ButtonText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Base">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Window">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>46</red>
       <green>52</green>
       <blue>54</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Shadow">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="AlternateBase">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>23</red>
       <green>26</green>
       <blue>27</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="ToolTipBase">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>220</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="ToolTipText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="PlaceholderText">
     <brush brushstyle="SolidPattern">
      <color alpha="128">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
   </active>
   <inactive>
    <colorrole role="WindowText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Button">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>46</red>
       <green>52</green>
       <blue>54</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Light">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>69</red>
       <green>78</green>
       <blue>81</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Midlight">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>57</red>
       <green>65</green>
       <blue>67</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Dark">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>23</red>
       <green>26</green>
       <blue>27</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Mid">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>30</red>
       <green>34</green>
       <blue>36</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Text">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="BrightText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="ButtonText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Base">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Window">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>46</red>
       <green>52</green>
       <blue>54</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Shadow">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="AlternateBase">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>23</red>
       <green>26</green>
       <blue>27</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="ToolTipBase">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>220</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="ToolTipText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="PlaceholderText">
     <brush brushstyle="SolidPattern">
      <color alpha="128">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
   </inactive>
   <disabled>
    <colorrole role="WindowText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>23</red>
       <green>26</green>
       <blue>27</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Button">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>46</red>
       <green>52</green>
       <blue>54</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Light">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>69</red>
       <green>78</green>
       <blue>81</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Midlight">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>57</red>
       <green>65</green>
       <blue>67</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Dark">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>23</red>
       <green>26</green>
       <blue>27</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Mid">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>30</red>
       <green>34</green>
       <blue>36</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Text">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>23</red>
       <green>26</green>
       <blue>27</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="BrightText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="ButtonText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>23</red>
       <green>26</green>
       <blue>27</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Base">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>46</red>
       <green>52</green>
       <blue>54</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Window">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>46</red>
       <green>52</green>
       <blue>54</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="Shadow">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="AlternateBase">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>46</red>
       <green>52</green>
       <blue>54</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="ToolTipBase">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>220</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="ToolTipText">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </colorrole>
    <colorrole role="PlaceholderText">
     <brush brushstyle="SolidPattern">
      <color alpha="128">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </colorrole>
   </disabled>
  </palette>
 </property>
  <property name="windowTitle">
   <string>Control setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_Control">
     <property name="title">
      <string>Selecting the desired control device</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="CheckBoxKeyBoard">
          <property name="text">
           <string>KeyBoard</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="CheckBoxMouse3d">
          <property name="text">
           <string>Mouse3d</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="CheckBoxJoystickLogitech">
          <property name="text">
           <string>Joystick Logitech</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
